src/http/client.rs

```diff
@@ -200,53 +188,20 @@ fn reason_into_header(reason: &str) -> Headers {
 
 /// **Note**: For all member functions that return a [`Result`], the
 /// Error kind will be either [`Error::Http`] or [`Error::Json`].
+#[derive(Debug)]
 pub struct Http {
     pub(crate) client: Client,
-    pub ratelimiter: Ratelimiter,
-    pub ratelimiter_disabled: bool,
-    pub proxy: Option<Url>,
+    pub ratelimiter: Option<Ratelimiter>,
+    pub api: String,
+    pub status_api: String,
     pub token: String,
     application_id: AtomicU64,
 }
 
-impl fmt::Debug for Http {
```

Comment:
This manual `Debug` implementation exists to hide the `token` in the debug representation to prevent someone accidentally leaking it. This shouldn't be removed.

Reply:
Ohh I see. Thank you, I'll add it back with a comment so future readers don't fall into the same trap as me